src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -428,12 +428,43 @@
 }
 
 .App:not(.oldDesign) .gameConfigContainer {
-  display: flex;
-  padding: 2vh 2vw;
-  height: 100%;
-  gap: min(3vw, 50px);
+  display: grid;
+  position: relative;
```

Comment:
Now that we have a `position: relative;` here, some images are slightly offset to the right, because background image is `width: 100vw;`. I'd suggest changing this to 100% in order to keep it centered

(disabled filters for better visibility in the example)

100vw
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/62100117/bc1bdf86-c433-4203-84eb-d5fb07df6cc4)

100%
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/62100117/f19b2585-1e19-4d3c-a617-2bf410208b8a)



Whole image:
![image](https://images.gog.com/62d0f061627c05c819adf950ac391c4aae76bacf121ee2dd1149d1cd9fe237a0.webp?namespace=gamesdb)

Reply:
I will change it to `100%` for both width and height since when viewport is <= 600px, then 100vh <= 100% and the image does not cover the whole background

Reply:
Fixed!